server/insert.sql

```diff
@@ -1,16 +1,3 @@
--- Insert statements
-INSERT INTO Role VALUES('Marketing Manager', 77000);
```

Comment:
do we not need this anymore?

Reply:
Nope, the Role and Duration tables were only added for us to normalize something.
Added another commit that removes the schemas as well